apps/whale-api/src/module.indexer/rpc.block.provider.ts

```diff
@@ -149,7 +152,11 @@ export class RPCBlockProvider {
       await this.indexer.invalidate(hash)
       await this.statusMapper.put(hash, height, Status.INVALIDATED)
     } catch (err) {
-      await this.statusMapper.put(hash, height, Status.ERROR)
+      if (err instanceof NotFoundIndexerError) {
```

Comment:
fix for (composite)poolswap index 
```suggestion
      if (err instanceof NotFoundIndexerError || err instanceof IndexerError) {
```